salt/modules/useradd.py

```diff
@@ -215,13 +216,16 @@ def add(
     usergroup
         Create and add the user to a new primary group of the same name
 
+    local (Only on systems with luseradd available)
+        Specifically add the user locally rather than possibly through remote providers (e.g. LDAP)
```

Comment:
```suggestion
        Specifically add the user locally rather than possibly through remote providers (e.g. LDAP)
        
        .. versionadded:: 3007.0
```